arch/nrf52/nrf52.ini

```diff
@@ -4,7 +4,7 @@ platform = platformio/nordicnrf52@^10.7.0
 extends = arduino_base
 platform_packages =
   ; our custom Git version until they merge our PR
-  framework-arduinoadafruitnrf52 @ https://github.com/geeksville/Adafruit_nRF52_Arduino.git
+  framework-arduinoadafruitnrf52 @ https://github.com/geeksville/Adafruit_nRF52_Arduino.git#4f591d0f71f75e5128fab9dc42ac72f1696cf89f
   toolchain-gccarmnoneeabi@~1.90301.0
```

Comment:
I think this can go to master pretty soon. So some clean up.
Do we still need this reference? Or can we go with the latest?